test/tests/val-2e/val-2ea.i

```diff
@@ -0,0 +1,316 @@
+# Validation Problem #2ea from TMAP7's V&V document
```

Comment:
The three input files have a lot in common, and there is a lot of redundancy, which makes maintenance harder. 

Please create a base input file that contains all the information/blocks that are common across the three input files, and the use the `!include val-2e-base` command in the input files for val-2ea, val-2ed, and val-2ee, which should by then be much shorter. 

Let me know if you have any questions about how to do that. 

Once you have done that, make sure to update the documentation to explain how the input files are built.